code/modules/admin/verbs/onlyone.dm

```diff
@@ -25,5 +25,8 @@ GLOBAL_VAR_INIT(highlander, FALSE)
 	log_admin("[key_name(usr)] used delayed THERE CAN BE ONLY ONE.")
 	addtimer(CALLBACK(src, .proc/only_one), 420)
 
-/mob/living/carbon/human/proc/make_scottish()
-	mind.add_antag_datum(/datum/antagonist/highlander)
\ No newline at end of file
+/mob/living/carbon/human/proc/make_scottish(super = FALSE)
+	if (super)
+		mind.add_antag_datum(/datum/antagonist/highlander/supermatter)
+	else
+		mind.add_antag_datum(/datum/antagonist/highlander)

```

Comment:
Have you tested if this works correctly with latejoins?

Reply:
it doesnt work with ghost roles for sure